extensions/README.md

```diff
@@ -102,3 +102,55 @@ To test extensions for a Simple site in Calypso, sandbox the simple site URL (ex
 If you need to modify the build process, bear in mind that config files are also
 synced to WordPress.com via Fusion. Consult with a Jetpack crew member to ensure
 you test the new build in both environments.
+
+## Good to know when developing Gutenberg extensions
+
+### Don't worry about dependencies
+
+The build takes care of core dependencies for both editor and view -scripts. React, lodash and `@wordpress/*` [dependencies](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md) are externalised and automatically enqueued in PHP for your extension.
+
+Extensions _always_ get [Gutenberg's polyfill scripts](https://github.com/WordPress/gutenberg/blob/master/docs/contributors/scripts.md#polyfill-scripts) enqueued so you can safely use methods not supported by older browsers such as IE11.
+
+### Jetpack sidebar
```

Comment:
:wave: FYI @codebykat if you wanna have a look at this section and have ideas for improvement. :-)